LAKMobile/src/components/AppText.tsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 import { StyleSheet, Text, TextProps } from 'react-native';
 
+<<<<<<< HEAD
```

Comment:
@aworld1 Seems like when you were merging main into this branch, some conflicts were not resolved (most likely in commit commit hash [c652d89](https://github.com/TritonSE/LAK-Goods-Transport-Application/pull/45/commits/c652d8921d001265577c1a0cc95b801084914ae4)) so make sure to fix those before merging to main otherwise the frontend on main won't compile. 

If this PR **PURELY** consists of backend changes, would recommend using the following command to reset the entire frontend files to the main branch. Make sure you have the most recent main pulled while doing this.

```
git checkout main -- LAKMobile/
```